wallet/network/store/v1/store.go

```diff
@@ -19,6 +20,15 @@ type Store struct {
 	networksHome string
 }
 
+type networkFileContent struct {
+	Name        string              `json:"name" toml:"-"`
```

Comment:
This property can go away now, since it no longer lives in the file

Reply:
Ah sorry, I was being too quick. Its gone now.